docs/cog_guides/admin.rst

```diff
@@ -0,0 +1,380 @@
+**Syntax**
+
+.. code-block:: none
+
+    [p]selfrole <selfrole>
+
+**Description**
+
+Add a role to yourself. It must have been configured as user settable
+by admins using the :ref:`selfrole add command <admin-command-selfrole-add>`.
+
+**Arguments**
+
+* ``<selfrole>``: The role you want to attribute to yourself. |role-input|
+
+.. _admin-command-selfrole-remove:
+
+"""""""""""""""
+selfrole remove
+"""""""""""""""
+
+**Syntax**
+
+.. code-block:: none
+
+    [p]selfrole remove <role>
+
+**Description**
+
+Remove a role from yourself. It must have been configured as user settable
+by admins.
+
+**Arguments**
+
+* ``<selfrole>``: The role you want to remove from yourself. |role-input|
+
+.. _admin-command-selfrole-list:
+
+"""""""""""""
+selfrole list
+"""""""""""""
+
+**Syntax**
+
+.. code-block:: none
+
+    [p]selfrole list
+
+**Description**
+
+List all of the available roles you can assign to yourself.
+
+.. _admin-command-selfrole-add:
+
+""""""""""""
+selfrole add
+""""""""""""
+
+.. note:: This command is locked to the members with the ``Manage roles``
+    permission.
+
+**Syntax**
+
+.. code-block:: none
+
+    [p]selfrole add <role>
+
+**Description**
+
+Add a role to the list of selfroles.
+
+.. warning:: Members will be able to assign themselves the role.
+    Make sure it doesn't give extra perms or anything that can break
+    your server's security.
+
+**Arguments**
+
+* ``<role>``: The role to add to the list. |role-input|
+
+.. _admin-command-selfrole-delete:
+
+"""""""""""""""
+selfrole delete
+"""""""""""""""
+
+.. note:: This command is locked to the member with the ``Manage roles``
+    permission.
+
+**Syntax**
+
+.. code-block:: none
+
+    [p]selfrole add <role>
+
+**Description**
+
+Removes a role from the list of selfroles.
+
+**Arguments**
+
+* ``<role>``: The role to remove from the list. |role-input|
+
+.. _admin-command-addrole:
+
+^^^^^^^
+addrole
+^^^^^^^
+
+.. note:: |admin-lock|
+
+**Syntax**
+
+.. code-block:: none
+
+    [p]addrole <rolename> [user]
+
+**Description**
+
+Adds a role to a member. If ``user`` is not given, it will be considered
+as yourself, the command author.
+
+**Arguments**
+
+* ``<role>``: The role to add to the member. |role-input|
+
+* ``[user=ctx]``: The member you want to add the role to. Defaults to the
+  command author. |member-input|
+
+.. _admin-command-removerole:
+
+^^^^^^^^^^
+removerole
+^^^^^^^^^^
+
+.. note:: |admin-lock|
+
+**Syntax**
+
+.. code-block:: none
+
+    [p]removerole
+
+**Description**
+
+Removes a role from a member. If ``user`` is not given, it will be considered
+as yourself, the command author.
+
+**Arguments**
+
+* ``<role>``: The role to remove. |role-input|
+
+* ``[user=ctx]``: The member to remove the role from. |member-input| Defaults to
+  the command author.
+
+.. _admin-command-editrole:
+
+^^^^^^^^
+editrole
+^^^^^^^^
+
+.. note:: |admin-lock|
+
+**Syntax**
+
+.. code-block:: none
+
+    [p]editrole
+
+**Description**
+
+Edits settings of a role.
```

Comment:
"Edits **the** settings of a role." is more grammatically correct, and you also used `the` in the same context in lines 231 and 253